cdap-ui/server/config/parser.js

```diff
@@ -71,7 +66,7 @@ function extractConfig(param) {
       path = getConfigPath(param);
       if (path && path.length) {
         path = path.replace(/\"/g, '');
-        cache[param] = require(path);
+        cache[param] = __non_webpack_require__(path);
```

Comment:
This is kind of awkward here. Can we not use require.resolve? Webpack documentation seems to suggest `require.resolve` to be a synchronous resolution for module check.

Reply:
I have to try it out, but from documentation, it doesn't seem to be possible:
`Synchronously retrieve a module's ID. The compiler will ensure that the dependency is available in the output bundle.`

From what I understand, this means that the path will actually gets resolved during build time instead of during runtime.